code/game/machinery/autolathe/artist_bench.dm

```diff
@@ -133,43 +133,60 @@
 			"magnum" = 8 + weight_vigilance,
 			"shotgun" = 8 + weight_robustness,
 			"rifle" = 8 + weight_vigilance,
-			"cap" = 16 + weight_biology,
+			"sniper" = 8 + max(weight_vigilance, weight_cognition),
+			"rocket" = 8 + weight_toughness
 		))
 
 		switch(gun_pattern)
 
 			if("pistol") //From havelock.dm, Arbitrary Values
 				R.caliber = pick(CAL_PISTOL)
-				R.damage_multiplier = 1.2 + rand(-5,5)/10
-				R.penetration_multiplier = 1.2 + rand(-5,5)/10
-				R.recoil_buildup = 18 + rand(-3,3)
+				R.damage_multiplier = 1.4 + rand(-5,5)/10
+				R.penetration_multiplier = 1.4 + rand(-5,5)/10
+				R.recoil_buildup = 3 + rand(-3,3)
 
 			if("magnum") //From consul.dm, Arbitrary values
 				R.caliber = CAL_MAGNUM
-				R.damage_multiplier = 1.2 + rand(-5,5)/10
-				R.penetration_multiplier = 1.2 + rand(-5,5)/10
-				R.recoil_buildup = 35 + rand(-5,5)
```

Comment:
If you change the base recoil also change the amount randomized. 35 +- 5 is not much, but 6 +- 5 is a lot